### ISSUE TITLE
Org-roam crashing Emacs (Windows 11)

### ISSUE THREAD

**Issue by raylemon** [org-roam/org-roam#2196]
### Description
Org-roam will not start (emacsql-timeout "Query timed out" 30)

#### Steps to Reproduce
Starting Doom Emacs after a fresh install

#### Backtrace
Debugger entered--Lisp error: (emacsql-timeout "Query timed out" 30)
  signal(emacsql-timeout ("Query timed out" 30))
  #f(compiled-function (connection &optional timeout) "Block until CONNECTION is waiting for further input." #<bytecode 0x1704a54c4c9a0eb8>)(#<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc>)
  apply(#f(compiled-function (connection &optional timeout) "Block until CONNECTION is waiting for further input." #<bytecode 0x1704a54c4c9a0eb8>) #<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc> nil)
  emacsql-wait(#<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc>)
  #f(compiled-function (#<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc> (:file "E:\\org-roam\15oam.db"))
  apply(#f(compiled-function 
  #f(compiled-function (&rest args) #<bytecode -0x109535e35d25036f>)(#<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc> (:file "E:\\org-roam\15oam.db"))
  apply(#f(compiled-function (&rest args) #<bytecode -0x109535e35d25036f>) #<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc> (:file "E:\\org-roam\15oam.db"))
  initialize-instance(#<emacsql-sqlite-connection emacsql-sqlite-connection-9737af80dc> (:file "E:\\org-roam\15oam.db"))
  #f(compiled-function (class &rest slots) "Default constructor for CLASS `eieio-default-superclass'.\nSLOTS are the initialization slots used by `initialize-instance'.\nThis static method is called when an object is constructed.\nIt allocates the vector used to represent an EIEIO object, and then\ncalls `initialize-instance' on that object." #<bytecode 0x86aa4d257a58e41>)(emacsql-sqlite-connection :file "E:\\org-roam\15oam.db")
  apply(#f(compiled-function (class &rest slots) "Default constructor for CLASS `eieio-default-superclass'.\nSLOTS are the initialization slots used by `initialize-instance'.\nThis static method is called when an object is constructed.\nIt allocates the vector used to represent an EIEIO object, and then\ncalls `initialize-instance' on that object." #<bytecode 0x86aa4d257a58e41>) emacsql-sqlite-connection (:file "E:\\org-roam\15oam.db"))
  make-instance(emacsql-sqlite-connection :file "E:\\org-roam\15oam.db")
  emacsql-sqlite("E:\\org-roam\15oam.db")
  org-roam-db()
  org-roam-db-sync()
  org-roam-db-autosync-mode(1)
  org-roam-setup()
  load-with-code-conversion("c:/Users/big04/.doom.d/config.el" "c:/Users/big04/.doom.d/config.el" t t)
  load("c:/Users/big04/.doom.d/config" t nomessage)
  (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage))
  (condition-case e (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name "config" doom-private-dir) doom-private-dir)))
  (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name "config" doom-private-dir) doom-private-dir))) (if custom-file (progn (load custom-file 'noerror (not doom-debug-mode)))))
  (progn (if doom-debug-p (progn (let ((inhibit-message t)) (message #("DOOM Initializing user config" 0 5 (face font-lock-comment-face)))))) (maphash (doom-module-loader doom-module-init-file) doom-modules) (doom-run-hooks 'doom-before-init-modules-hook) (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name "config" doom-private-dir) doom-private-dir))) (if custom-file (progn (load custom-file 'noerror (not doom-debug-mode))))))
  (if init-p (progn (if doom-debug-p (progn (let ((inhibit-message t)) (message #("DOOM Initializing user config" 0 5 (face font-lock-comment-face)))))) (maphash (doom-module-loader doom-module-init-file) doom-modules) (doom-run-hooks 'doom-before-init-modules-hook) (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name "config" doom-private-dir) doom-private-dir))) (if custom-file (progn (load custom-file 'noerror (not doom-debug-mode)))))) nil)
  (let* ((init-p (and t (condition-case e (let (file-name-handler-alist) (load (expand-file-name doom-module-init-file doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name doom-module-init-file doom-private-dir) doom-private-dir)))))) (if init-p (progn (if doom-debug-p (progn (let ((inhibit-message t)) (message #("DOOM Initializing user config" 0 5 ...))))) (maphash (doom-module-loader doom-module-init-file) doom-modules) (doom-run-hooks 'doom-before-init-modules-hook) (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let (file-name-handler-alist) (load (expand-file-name "config" doom-private-dir) t 'nomessage)) ((debug doom-error) (signal (car e) (cdr e))) ((debug error) (doom--handle-load-error e (expand-file-name "config" doom-private-dir) doom-private-dir))) (if custom-file (progn (load custom-file 'noerror (not doom-debug-mode)))))) nil))
  (progn (setq doom-init-modules-p t) (if no-config-p nil (if doom-debug-p (progn (let ((inhibit-message t)) (message #("DOOM Initializing core modules" 0 5 (face font-lock-comment-face)))))) (doom-initialize-core-modules)) (let* ((init-p (and t (condition-case e (let (file-name-handler-alist) (load ... t ...)) ((debug doom-error) (signal ... ...)) ((debug error) (doom--handle-load-error e ... doom-private-dir)))))) (if init-p (progn (if doom-debug-p (progn (let (...) (message ...)))) (maphash (doom-module-loader doom-module-init-file) doom-modules) (doom-run-hooks 'doom-before-init-modules-hook) (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let (file-name-handler-alist) (load ... t ...)) ((debug doom-error) (signal ... ...)) ((debug error) (doom--handle-load-error e ... doom-private-dir))) (if custom-file (progn (load custom-file ... ...))))) nil)))
  (if (or force-p (not doom-init-modules-p)) (progn (setq doom-init-modules-p t) (if no-config-p nil (if doom-debug-p (progn (let ((inhibit-message t)) (message #("DOOM Initializing core modules" 0 5 ...))))) (doom-initialize-core-modules)) (let* ((init-p (and t (condition-case e (let ... ...) (... ...) (... ...))))) (if init-p (progn (if doom-debug-p (progn (let ... ...))) (maphash (doom-module-loader doom-module-init-file) doom-modules) (doom-run-hooks 'doom-before-init-modules-hook) (if no-config-p nil (maphash (doom-module-loader doom-module-config-file) doom-modules) (doom-run-hooks 'doom-init-modules-hook) (condition-case e (let ... ...) (... ...) (... ...)) (if custom-file (progn ...)))) nil))))
  doom-initialize-modules()
  load-with-code-conversion("c:/Users/big04/.emacs.d/init.el" "c:/Users/big04/.emacs.d/init.el" nil t)
  command-line-1(("-l" "..\\init.el" "-f" "doom-run-all-startup-hooks-h"))
  command-line()
  normal-top-level()

#### Expected Results

Doom Emacs running nicely

### Environment
Can’t use M-x
Windows 11
Emacs 28.1 with native-comp
Org-roam v2

**Comment by nobiot** [comment]
This? https://github.com/org-roam/org-roam/wiki/V2-Binary-emacsql-sqlite.exe-for-Windows

**Comment by raylemon** [comment]
No, I have already replace my version with yours. It don’t work. I think it’s because I use Doom Emacs. With vanilla Emacs, all is fine…

**Comment by nobiot** [comment]
Then, it’s better to raise this issue there. I don’t think Jethro and the project can address Doom specific issues
